dev/com.ibm.ws.concurrent.persistent_fat/fat/src/com/ibm/ws/concurrent/persistent/fat/PersistentExecutorWithFailoverEnabledTest.java

```diff
@@ -33,168 +31,135 @@
-        runTest(server, APP_NAME, "testBlockAfterCancelByNameFE");
-    }
-
-    @Test
-    public void testBlockAfterFindByIdFE() throws Exception {
-        runTest(server, APP_NAME, "testBlockAfterFindByIdFE");
-    }
-
-    @Test
-    public void testBlockAfterFindByNameFE() throws Exception {
-        runTest(server, APP_NAME, "testBlockAfterFindByNameFE");
-    }
-
-    @Test
-    public void testBlockAfterRemoveByIdFE() throws Exception {
-        runTest(server, APP_NAME, "testBlockAfterRemoveByIdFE");
-    }
-
-    @Test
-    public void testBlockAfterRemoveByNameFE() throws Exception {
-        runTest(server, APP_NAME, "testBlockAfterRemoveByNameFE");
-    }
-
-    @Test
-    public void testBlockAfterScheduleFE() throws Exception {
-        runTest(server, APP_NAME, "testBlockAfterScheduleFE");
-    }
-
-    @Test
-    public void testBlockRunningTaskFE() throws Exception {
-        runTest(server, APP_NAME, "testBlockRunningTaskFE");
-    }
-
-    @Test
-    public void testBlockRunningTaskThatCancelsSelfFE() throws Exception {
-        runTest(server, APP_NAME, "testBlockRunningTaskThatCancelsSelfFE");
-    }
-
-    @Test
-    public void testBlockRunningTaskThatRemovesSelfFE() throws Exception {
-        runTest(server, APP_NAME, "testBlockRunningTaskThatRemovesSelfFE");
-    }
-
-    @Test
-    public void testCancelRunningTaskFE() throws Exception {
-        runTest(server, APP_NAME, "testCancelRunningTaskFE");
-    }
-
-    @Test
-    public void testRemoveRunningTaskAutoPurgeFE() throws Exception {
-        runTest(server, APP_NAME, "testRemoveRunningTaskAutoPurgeFE");
-    }
-
-    @Test
-    public void testRemoveRunningTaskFE() throws Exception {
-        runTest(server, APP_NAME, "testRemoveRunningTaskFE");
-    }
+	private static final String APP_NAME = "schedtest";
+
+	@ClassRule
+	public static RepeatTests r = RepeatTests.withoutModification()//
+			.andWith(new JakartaEE9Action());
+
+	@Server("com.ibm.ws.concurrent.persistent.fat.failover")
+	@TestServlet(servlet = SchedulerFATServlet.class, path = APP_NAME)
+	public static LibertyServer server;
+	
+	public static final JdbcDatabaseContainer<?> testContainer = DatabaseContainerFactory.create();
+
+	@BeforeClass
+	public static void setUp() throws Exception {
+		testContainer.start();
+		
+		server.addEnvVar("DB_DRIVER", DatabaseContainerType.valueOf(testContainer).getDriverName());
+
+		DatabaseContainerUtil.setupDataSourceProperties(server, testContainer);
+
+		ShrinkHelper.defaultDropinApp(server, APP_NAME, "web");
+
+		// Use the Jakarta version of test features if Jakarta is being used.
+		if (JakartaEE9Action.isActive()) {
+			ServerConfiguration config = server.getServerConfiguration();
+			Set<String> features = config.getFeatureManager().getFeatures();
+			features.remove("persistentexecutor-1.0");
+			features.remove("timerinterfacestestfeature-1.0");
+			features.add("persistentExecutor-2.0");
+			features.add("timerInterfacesTestFeature-2.0");
```

Comment:
Shouldn't the JakartaEE9Action do this automatically?  Or is it because it the features switched from all lowercase to camel case?

Reply:
Partially yes -- we could update the `JakartaEE9Action` to auto-update the `persistentExecutor` feature, but we shouldn't add the `timerInterfacesTestFeature` to the EE9 action since that is a test feature only used in this bucket.